subsys/bluetooth/services/fast_pair/fp_crypto/include/fp_crypto.h

```diff
@@ -103,6 +109,72 @@ size_t fp_crypto_account_key_filter_size(size_t n);
 int fp_crypto_account_key_filter(uint8_t *out, const struct fp_account_key *account_key_list,
 				 size_t n, uint8_t salt);
 
+/** Encrypt data using AES-CTR.
+ *
+ * @param[out] out Buffer to receive encrypted data.
+ * @param[in] in Plaintext data.
+ * @param[in] data_len Length of input and output data (in bytes).
+ * @param[in] key 128-bit (16-byte) AES key.
+ * @param[in] nonce 64-bit (8-byte) nonce.
+ *
+ * @return 0 If the operation was successful. Otherwise, a (negative) error code is returned.
+ */
+int fp_crypto_aes_ctr_encrypt(uint8_t *out, const uint8_t *in, size_t data_len, const uint8_t *key,
+			      const uint8_t *nonce);
+
+/** Decrypt data using AES-CTR.
+ *
+ * @param[out] out Buffer to receive plaintext data.
+ * @param[in] in Ciphertext data.
+ * @param[in] data_len Length of input and output data (in bytes).
+ * @param[in] key 128-bit (16-byte) AES key.
+ * @param[in] nonce 64-bit (8-byte) nonce used to encrypt data.
+ *
+ * @return 0 If the operation was successful. Otherwise, a (negative) error code is returned.
+ */
+int fp_crypto_aes_ctr_decrypt(uint8_t *out, const uint8_t *in, size_t data_len, const uint8_t *key,
+			      const uint8_t *nonce);
+
+/** Generate HMAC-SHA256.
+ *
+ * @param[out] out 256-bit (32-byte) buffer to receive hashed result.
+ * @param[in] in Input data.
+ * @param[in] data_len Length of input data.
+ * @param[in] aes_key 128-bit (16-byte) AES key used to encrypt data.
+ *
+ * @return 0 If the operation was successful. Otherwise, a (negative) error code is returned.
+ */
+int fp_crypto_hmac_sha256(uint8_t *out, const uint8_t *in, size_t data_len, const uint8_t *aes_key);
```

Comment:
Please move this function below fp_crypto_sha256() as they are in the same category (hashing functions)